src/NCI.OCPL.Api.Glossary/Controllers/TermsController.cs

```diff
@@ -18,13 +19,27 @@ namespace NCI.OCPL.Api.Glossary.Controllers
     [ApiController]
     public class TermsController : Controller
     {
+        private readonly ILogger _logger;
         private readonly ITermsQueryService _termsQueryService;
 
+        private readonly LinkedList<Tuple<string, AudienceType>> _fallbackOptions = new LinkedList<Tuple<string, AudienceType>>(
+            new Tuple<string, AudienceType>[]
+            {
+                new Tuple<string, AudienceType>("Cancer.gov", AudienceType.Patient),
+                new Tuple<string, AudienceType>("Cancer.gov", AudienceType.HealthProfessional),
+                new Tuple<string, AudienceType>("NotSet", AudienceType.Patient),
+                new Tuple<string, AudienceType>("NotSet", AudienceType.HealthProfessional),
+                new Tuple<string, AudienceType>("Genetics", AudienceType.Patient),
+                new Tuple<string, AudienceType>("Genetics", AudienceType.HealthProfessional)
+            }
+        );
+
```

Comment:
This fallback order is only correct for summaries with an audience of "Patient".  There is a different order for Health Professional, Genetic summaries, and a third for all other summaries with a "Health Professional"  audience.

The full set of fullback orders is in the wiki, starting at: https://github.com/NCIOCPL/glossary-api/wiki/Glossary-Term-Display-Business-Rules#patient-summaries.

Reply:
@blairlearn Take a look at the logic. The order of choices is the same for each type, just where you start is different...

Reply:
You are correct in a way. The difference is between Patient and HP.  Both HP, Genetics; and HP Notset follow the same loop.

Reply:
Yes.  That's reflected in my comments further down.  The bit where things loop around is definitely clever though.